subprojects/core/src/test/groovy/org/gradle/util/NameValidatorTest.groovy

```diff
@@ -42,7 +42,7 @@ class NameValidatorTest extends Specification {
     @Shared
     def domainObjectContainersWithValidation = [
         ["artifact types", new DefaultArtifactTypeContainer(DirectInstantiator.INSTANCE, TestUtil.attributesFactory())],
-        ["configurations", new DefaultConfigurationContainer(null, DirectInstantiator.INSTANCE, domainObjectContext(), Mock(ListenerManager), null, null, null, null, Mock(FileCollectionFactory), null, null, null, null, null, TestUtil.attributesFactory(), null, null)],
+        ["configurations", new DefaultConfigurationContainer(null, DirectInstantiator.INSTANCE, domainObjectContext(), Mock(ListenerManager), null, null, null, null, Mock(FileCollectionFactory), null, null, null, null, null, TestUtil.attributesFactory(), null, null, null)],
```

Comment:
`null, null, null, null, null`... this is some horrible stuff here. :/

Reply:
Yeah, `DefaultConfigurationContainer` has way to many dependencies :(

Do you think mocking more services would improve something here? Then we could pass those instead of `null`.